chain/consensus/mir/manager.go

```diff
@@ -138,27 +139,22 @@ func NewManager(ctx context.Context, addr address.Address, h host.Host, api v1ap
 		InitialValidatorSet: initialValidatorSet,
 		ToMir:               make(chan chan []*mirproto.Request),
 		checkpointRepo:      cfg.CheckpointRepo,
+		segmentLength:       1,
```

Comment:
Why do we set `segmentLength` to 1? Are we removing the ability to configure the checkpoint period?

Reply:
Yes, we did! We discussed that with you at the beginning of December!
We agreed that SegmentLength is the constant within the network.

Reply:
What I meant is that we should set `segmentLength` as the parameter to influence the checkpoint period (not to make it a constant). Sorry if I didn't explain myself right. Let's add `segmentLength` as param and `1` as its default value. Thanks!

Reply:
Fixed - 40eb43e7a66f3824cf1adb93fb1baf470ec47028